src/components/SelectInput/SearchSelectInput.tsx

```diff
@@ -25,7 +25,7 @@ function Option(props: OptionProps) {
                 <MdCheck />
             </div>
             <div className={styles.label}>
-                { children }
+                {children}
```

Comment:
```suggestion
                { children }
```

Reply:
I consulted this with safar dai and he recommended that removing the spaces would be better.